src/test/java/com/fittracker/fittracker/dataprovider/TestValues.java

```diff
@@ -0,0 +1,17 @@
+package com.fittracker.fittracker.dataprovider;
+
+import java.time.LocalDate;
+import java.util.UUID;
+
+public class TestValues {
```

Comment:
I think this is overkill, the purpose of this ticket was to simplify entity creation in tests, not to extract every reused test value. Let's get rid of this class and just hardcode the values in data providers (the way it's done in `registerResponse()`) and in test classes. And if the same values are used multiple times in a single test class - we can just have it as a private static final variable on that class.